reference/sisdata.v1.yaml

```diff
@@ -245,23 +253,26 @@ components:
         subjectCode:
           type: string
           description: 'Officiële elementcode, bron: Ministerie van OCW'
+        schoolSubjectName:
+          type: string
+          description: Optional name of subject if different within specific school.
         level:
           type: string
           description: 'havo, vwo, etc.'
         student:
           $ref: '#/components/schemas/student'
-        class:
-          $ref: '#/components/schemas/class'
+        group:
+          $ref: '#/components/schemas/group'
         school:
           $ref: '#/components/schemas/school'
+        schoolYear:
+          type: string
+          description: A reference to the school year for this group.
```

Comment:
I'm not sure we should add this property, as mentioned in https://github.com/stichtingsem/technology-prototype/issues/28#issuecomment-675873495

Nitpicking: If we do decide to add this property the word 'group' in the description should be changed to 'subjectChoice' I think

Reply:
Done